kubelet/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "7.0.0",
+  "min_agent_version": "6.0.0",
+  "name": "kubelet",
+  "short_description": "Collects container stats from kubelet.",
+  "guid": "55039e21-7e89-41fb-968c-ab8bf8f25da0",
+  "support": "core",
+  "supported_os": ["linux"],
+  "version": "1.0.0"
```

Comment:
`version` is not in the manifest anymore since version 1.0.0, can you remove it?